utils/messages.js

```diff
@@ -1,62 +1,162 @@
-	 */
-	greetingMessage: function(name, datetime, room) {
-		return "Hello from the Alaska State Court System. " +
-				"We found a case for " + name + " scheduled on " +
-				datetime.format('ddd, MMM Do') + " at " +
-				datetime.format("h:mm A") + ", at " + room +
-				". Would you like a courtesy reminder the day before? (reply YES or NO)";
-	},
-
-	/**
-	 * Message to send when we we cannot find a person's court case for too long.
-	 * @return {string} Not Found Message
-	 */
-	unableToFindCitationForTooLong: function() {
-        return "We haven\'t been able to find your court case. You can go to " + process.env.COURT_PUBLIC_URL + " for more information. - Alaska State Court System";
-	},
-
-	/**
-	 * Reminder message body
-	 *
-	 * @param  {Object} reminder reminder record.
-	 * @return {string} message body.
-	 */
-	reminder: function(reminder) {
-		return "Reminder: It appears you have a court hearing tomorrow at " +
-			dates.fromUtc(reminder.date).format("h:mm A") +
-        	" at " + reminder.room +
-        	". You should confirm your hearing date and time by going to " +
-        	process.env.COURT_PUBLIC_URL +
-        	". - Alaska State Court System";
-	},
-
-	/**
-	 * Send a twilio message
-	 *
-	 * @param  {string} to   phone number message will be sent to
-	 * @param  {string} from who the message is being sent from
-	 * @param  {string} body message to be sent
-	 * @param  {function} function for resolving callback
-	 * @return {Promise} Promise to send message.
-	 */
-	send: function(to, from, body) {
-		return new Promise(function(resolve, reject) {
-			client.sendMessage({to: to, from: from, body: body}, resolve)
-		})
-	}
+/**
+ * message to go to the site for more information
+ *
+ * @return {String} message.
+ */
+function forMoreInfo() {
+  return normalizeSpaces(`OK. You can always go to ${process.env.COURT_PUBLIC_URL}
+    for more information about your case and contact information.`);
+}
 
+/**
+ * tell them of the court date, and ask them if they would like a reminder
+ *
+ * @param  {Boolean} includeSalutation true if we should greet them
+ * @param  {string} name Name of cited person/defendant.
+ * @param  {moment} datetime moment object containing date and time of court appearance.
+ * @param  {string} room room of court appearance.
+ * @return {String} message.
+ */
+function foundItAskForReminder(includeSalutation, name, datetime, room) {
+  const salutation = `Hello from the ${process.env.COURT_NAME}. `;
 
+  const caseInfo = `We found a case for ${name} scheduled
+    ${(datetime.isSame(dates.now(), 'd') ? 'today' : `on ${datetime.format('ddd, MMM Do')}`)}
+    at ${datetime.format('h:mm A')}, at ${room}.`;
+
+  let futureHearing = '';
+  if ((datetime.diff(dates.now()) > 0) && (datetime.isSame(dates.now(), 'd'))) { // Hearing today
+    futureHearing = ' a future hearing';
+  } else if (datetime.diff(dates.now()) <= 0) { // Hearing already happened
+    futureHearing = ' a future hearing';
```

Comment:
This looks like it will append ' a future hearing' to messages about cases that already happened. Am I misreading that? It seems like that is in the current code too in web.js. Is this the correct behavior?

Reply:
@markmeyerphoto that's correct.  It is basically telling them that it will watch for future ones and let them know. It could be more explicit, but I didn't want to alter that part in this PR.